apps/hash-graph/lib/graph/src/shared/subgraph/query.rs

```diff
@@ -166,6 +166,10 @@ pub struct StructuralQuery<'p, R: Record> {
 }
 
 impl<'p, R: Record> StructuralQuery<'p, R> {
+    #[expect(
+        clippy::same_name_method,
+        reason = "This is supposed to replace the derived `schema` method"
+    )]
     fn schema() -> RefOr<Schema> {
```

Comment:
It's not implementing the trait though right? So do we maybe we want to just call it something else like `open_api_schema` or is it actually important it's called the same thing?

Reply:
The trait is implemented on the four type aliases. I will figure out if we can just implement the trait manually.

Reply:
I renamed the function